### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DominikDary**
## High-level description

Remove Windows Support from DC/OS open and EE

## Corresponding DC/OS tickets (required)

  - [D2IQ-67966](https://jira.d2iq.com/browse/D2IQ-67966) Remove Windows Support from DC/OS open and EE

### CODE REVIEW

**Comment by DominikDary** [comment]
@mesosphere-mergebot request-review  @jongiddy

File: README.md [hunk 1]
```diff
@@ -36,10 +36,7 @@ To find the git SHA of any given release, check the latest commit in the version
 |--------------|--------------------|
 | Latest Stable| `https://downloads.dcos.io/dcos/stable/dcos_generate_config.sh` |
 | Latest Master Linux| `https://downloads.dcos.io/dcos/testing/master/dcos_generate_config.sh` |
-| Latest Master Windows| `https://downloads.dcos.io/dcos/testing/master/windows/dcos_generate_config_win.sh` |
 | Latest Build of Specific PR Linux| `https://downloads.dcos.io/dcos/testing/pull/<github-pr-number>/dcos_generate_config.sh` |
```
Comment (jongiddy) [inline README.md:39]: Remove `Linux` from this table as well. So just `Latest Master` and `Latest Build of Specific PR`

Comment (DominikDary) [reply README.md]: done

**Comment by DominikDary** [comment]
@mesosphere-mergebot changelog-not-required md update

**Comment by DominikDary** [comment]
@mesosphere-mergebot label Ship It

**Comment by DominikDary** [comment]
@mesosphere-mergebot merge-it

**Pull request merged**